src/Query/Bool.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace tantrum_elastic\Query;
+
+use tantrum_elastic\Lib\Validate;
+use tantrum_elastic\Query\Lib\ClauseCollection;
+use tantrum_elastic\Query\Lib\MinimumShouldMatch;
+use tantrum_elastic\Query\Lib\Boost;
+
+/**
+ * Class Bool
+ * @package tantrum_elastic\Query
+ */
+class Bool extends Base
+{
+    use MinimumShouldMatch;
+    use Boost;
+
+    use Validate\Integers;
+    use Validate\Floats;
+
+    /**
+     * Return the must collection. Create one if it doesn't exist
+     *
+     * @return Collection
+     */
+    private function getMust()
```

Comment:
What uses these private methods? Can't be used outside this class, and the public methods don't seem to use them at all.